Modules/xSharePoint/DSCResources/MSFT_xSPOutgoingEmailSettings/MSFT_xSPOutgoingEmailSettings.psm1

```diff
@@ -0,0 +1,86 @@
+function Get-TargetResource
+{
+    [CmdletBinding()]
+    [OutputType([System.Collections.Hashtable])]
+    param
+    (
+        [parameter(Mandatory = $true)] [System.String] $WebAppUrl,
+        [parameter(Mandatory = $true)] [System.String] $SMTPServer,
+        [parameter(Mandatory = $true)] [System.String] $FromAddress,
+        [parameter(Mandatory = $true)] [System.String] $ReplyToAddress,
+        [parameter(Mandatory = $true)] [System.String] $CharacterSet
+        
+    )
+
+    Write-Verbose -Message "Retrieving outgoing email settings configuration "
+
+    $result = Invoke-xSharePointCommand -Arguments $PSBoundParameters -ScriptBlock {
```

Comment:
You need to add an InstallAccount parameter (look at pretty much any other resource we have to see how we do it), this is needed to ensure we can support PowerShell 4 where PsDscRunAsCredential is not available.